netapp-ontap-sql/scripts/InstallNetAppPS.ps1

```diff
@@ -0,0 +1,4 @@
+New-Item C:\NetApp -Type Directory
+$WebClient = New-Object System.Net.WebClient
+$WebClient.DownloadFile("https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/netapp-ontap-sql/scripts/NetApp_PowerShell_Toolkit_4.3.0.msi","C:\NetApp\NetApp_PowerShell_Toolkit_4.3.0.msi")
```

Comment:
is there an "official" download location for this - should be more resilient than the repo

Reply:
Official download location requires login to NetApp, this is discussed with NetApp and planned to keep here